docs/en/stack/security/authorization/built-in-roles.asciidoc

```diff
@@ -26,7 +30,6 @@ change between releases.
 * This role does not provide access to the beats indices and is not
 suitable for writing beats output to {es}.
 ===============================
-
```

Comment:
This line removal isn't in the original PR (#80) but I assume it doesn't break anything in the backport?